Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/DirectoryInitialization.java

```diff
@@ -145,7 +145,20 @@ private static void initializeExternalStorage(Context context)
     createWiimoteProfileDirectory(profileDirectory);
 
     copyAsset("GCPadNew.ini", new File(configDirectory, "GCPadNew.ini"), true, context);
-    copyAsset("WiimoteNew.ini", new File(configDirectory, "WiimoteNew.ini"), false, context);
+
+    SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(context);
+    if (!prefs.getBoolean("WiimoteNewV2", false))
```

Comment:
This is pretty ugly, but I guess we don't have much choice with the way the controller system currently works on Android...

Maybe consider having a "WiimoteNewVersion" integer instead of a boolean so it'll be a bit cleaner if we need to do the same thing again later?

Reply:
I guess in this instance it's ok to use an integer since it's an 'all or nothing' change. I usually use a bool in case a change should only apply once, such as the case in [InputOverlay](https://github.com/dolphin-emu/dolphin/blob/master/Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java#L1086) when setting the default button placements.